packages/common/src/events/edit.ts

```diff
@@ -28,7 +28,6 @@ export async function createHubEvent(
   // so set endDate to startDate
   event.endDate = event.startDate;
 
-  // TODO: how to handle slugs
   // TODO: how to handle events being discussable vs non-discussable
```

Comment:
Is this something that needs to be addressed on this PR, or is there another issue for it?

Reply:
Nice catch. No, we'll address on a future PR.